code/modules/projectiles/gun_attachables.dm

```diff
@@ -1311,7 +1311,8 @@ Defined in conflicts.dm of the #defines folder.
 	slot = "under"
 	fire_sound = 'sound/weapons/guns/fire/flamethrower3.ogg'
 	flags_attach_features = ATTACH_REMOVABLE|ATTACH_ACTIVATION|ATTACH_RELOADABLE|ATTACH_WEAPON
-	attachment_firing_delay = 35
+	attachment_firing_delay = 25
+	var/last_fired = 0		//When it was last fired, related to world.time.
```

Comment:
Use the COOLDOWN_DECLARE macro types for these cooldowns, see cooldown.dm

Reply:
Sure I will look at that